Tests/UnitTests/Mocks/MockAttributionFetcher.swift

```diff
@@ -16,8 +16,9 @@ class MockAttributionFetcher: AttributionFetcher {
     }
 
     var adServicesTokenCollectionCalled = false
+    var adServicesTokenToReturn: String? = "adServicesToken"
```

Comment:
I think the convention in other mocks is to call this `mockedAdServicesToken` or `stubbedAdServicesToken`.